File: food-order/Domain/Restaurant/Restaurant.cs
using System.Collections.Generic;

namespace food_order.Domain.Restaurant
{
    public class Restaurant
    {
        public Restaurant(long id, string uuid, string name, 
            string address, List<MenuItem> itens)
        {
            Id = id;
            Uuid = uuid;
            Name = name;
            Address = address;
            Itens = itens;
        }

        public long Id { get; }
        public string Uuid { get; }
        public string Name { get; }
        public string Address { get; }
        public List<MenuItem> Itens { get; }
    }
}

File: food-order/internal/app/domain/restaurant.go
package domain

import "github.com/google/uuid"

type Restaurant struct {
	Uuid uuid.UUID
	Name string
}

type RestaurantDetail struct {
	Uuid  uuid.UUID
	Name  string
	Items []MenuItem
}

type MenuItem struct {
	Uuid      uuid.UUID
	Name      string
	UnitValue float64
}
